File: shopping-client-angular/src/app/models/receivable.ts
import { Generic } from './generic';
import { Client } from './client';
import { Article } from './article';

export class Receivable implements Generic {
    id: number;
    orderId: number;
    invoiceId: string;
    totalIgv: string;
    totalInvoice: string;
    paymentDate: string;
    client: Client;
    articles: Article[];
}

File: order_processing/src/model/receivable.go
package model

type Receivable struct {
    OrderId    	  int64   	 `json:"orderId"`
	InvoiceId  	  string  	 `json:"invoiceId"`
	TotalIgv  	  string  	 `json:"totalIgv"`
	TotalInvoice  string  	 `json:"totalInvoice"`
	PaymentDate   string  	 `json:"paymentDate"`
	Client  	  Client  	 `json:"client"`
	Articles  	  []Article  `json:"articles"`
}